src/libs/ReportUtils.ts

```diff
@@ -3267,16 +3099,14 @@ function shouldReportBeInOptionList(report, currentReportId, isInGSDMode, betas,
 
 /**
  * Attempts to find a report in onyx with the provided list of participants. Does not include threads, task, money request, room, and policy expense chat.
- * @param {Array<Number>} newParticipantList
- * @returns {Array|undefined}
  */
-function getChatByParticipants(newParticipantList) {
-    const sortedNewParticipantList = _.sortBy(newParticipantList);
-    return _.find(allReports, (report) => {
+function getChatByParticipants(newParticipantList: number[]): OnyxEntry<Report> | undefined {
```

Comment:
```suggestion
function getChatByParticipants(newParticipantList: number[]): OnyxEntry<Report> {
```

Reply:
@kubabutkiewicz Append ` ?? null` to the return in order to get rid of this `undefined`